pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [project]
 name = "peval"
 version = "0.2.0.dev"
-license = "MIT"
+license = {text = "MIT"}
```

Comment:
What problems does it cause? I have this set in my other projects, and they published just fine only a few weeks ago.